RELEASE/scripts/autoscend/auto_util.ash

```diff
@@ -1381,8 +1381,8 @@ boolean cloverUsageInit()
 	//use a clover if we have one in inventory or closet
 	if(item_amount($item[11-Leaf Clover]) < 1)
 	{
-		//try to get one out of closet
-		retrieve_item(1, $item[11-Leaf Clover]);	
+		//try to get one out of closet, store the result to avoid an error being thrown
+		boolean result = retrieve_item(1, $item[11-Leaf Clover]);	
```

Comment:
I think we should make a function called `boolean getClover()` instead that attempts to get a clover and returns true if it manages to get one

Reply:
```suggestion
		catch retrieve_item(1, $item[11-Leaf Clover]);	
```